releases/v20.1.0.md

```diff
@@ -0,0 +1,129 @@
+**Admin UI Improvements** | The [**Network Latency**](../v20.1/admin-ui-network-latency-page.html) page of the Admin UI is now easier to access and has been redesigned to help you understand the round-trip latencies between all nodes in your cluster.<br><br>Also, the **Statement Details** page now allows you to write information about a SQL statement to a [diagnostics bundle](../v20.1/admin-ui-statements-page.html#diagnostics) that you can download. This bundle consists of a JSON file that contains a distributed trace of the SQL statement, a physical query plan, execution statistics, and other information about the query.
+**Slow Query Log** | You can now enable a [slow query log](../v20.1/query-behavior-troubleshooting.html#slow-query-log) to record SQL queries whose service latency exceeds a specified threshold.
+**Column Families in Secondary Indexes** | [Secondary indexes](../v20.1/column-families.html) now respect the column family definitions applied to tables. When you define a secondary index, CockroachDB breaks the secondary index key-value pairs into column families, according to the family and stored column configurations.
+
+### Enterprise features
+
+These features require an [enterprise license](../v19.2/enterprise-licensing.html). Register for a 30-day trial license [here](https://www.cockroachlabs.com/get-cockroachdb/), or consider testing enterprise features locally using the [`cockroach demo`](../v19.2/cockroach-demo.html) CLI command, which starts an in-memory CockroachDB cluster with a temporary enterprise license pre-loaded. [CockroachCloud clusters](https://cockroachlabs.cloud/) also include all enterprise features.
+
+Feature | Description
+--------|------------
+**Full-cluster backups** | You can now use CockroachDB's `BACKUP` feature to [back up an entire cluster's data](../v20.1/backup.html#backup-targets), including all configuration and system information such as [user privileges](../v20.1/authorization.html#privileges), [zone configurations](../v20.1/configure-replication-zones.html), and [cluster settings](../v20.1/cluster-settings.html). In rare disaster recovery situations, you can use the `RESTORE` feature to [restore a full cluster backup to a new cluster](../v20.1/restore.html#full-cluster). Restoring a full cluster backup to an existing cluster is not supported.
+**Encrypted backups** | You can now use an encryption key to [encrypt data in Enterprise `BACKUP` files](../v20.1/backup.html#encrypted-backups), and to [decrypt the data upon `RESTORE`](../v20.1/restore.html#restore-from-an-encrypted-backup).
```

Comment:
If we want to be really pedantic, the option is call `encryption_passphrase` so maybe "You can now use an encryption passphrase to..."?

Reply:
Done.